pkg/kubelet/apis/config/v1beta1/defaults.go

```diff
@@ -39,6 +39,8 @@ const (
 
 	// See https://github.com/kubernetes/enhancements/tree/master/keps/sig-node/2570-memory-qos
 	DefaultMemoryThrottlingFactor = 0.9
+
+	DefaultRecheckDuration = 24 * time.Hour
```

Comment:
a left over `DefaultRecheckDuration`. also can this be a bit more clear like 
`DefaultImagePullRecheckPeriod`? More verbose but since it's exported and used elsewhere I think that'll make it clearer

Reply:
Updated and add an link to the KEP here like above.